eventuals/BUILD.bazel

```diff
@@ -131,5 +131,6 @@ cc_library(
         ":events",
         ":http",
         "@com_github_3rdparty_stout//:stout",
+        "@com_github_expected//:expected",
```

Comment:
We don't need this just yet! And also, seeing that name name, let's update it to `com_github_tl_expected` in the other PR. Thanks!